docker-compose.yml

```diff
@@ -21,6 +28,48 @@ services:
       dockerfile: ./Dockerfile.pip
     volumes:
       - ./app:/app/app
+      - ./src:/app/src
     command: uvicorn app.main:app --host 0.0.0.0 --port 8000 --reload
+    depends_on:
+      - pgdb
     ports:
       - "8000:8000"
+    env_file:
+      - ./.env
+    networks:
+      - webnet
+
+  pgdb:
+    container_name: fastapidb-psqlcontainer
+    image: postgres
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DB}
+    volumes:
+      - fastapidb-postgres-data:/var/lib/postgresql/data
+    networks:
+      - webnet
+
+  pgadmin:
+    container_name: fastapidb-pgAdmin
+    image: dpage/pgadmin4
+    depends_on:
+      - pgdb
+    ports:
+      - "5050:80"
+    env_file:
+      - ./.env
+    volumes:
+      - fastapidb-pgAdmin-data:/var/lib/pgadmin
+    networks:
+      - webnet

```

Comment:
This looks cool, although i install one of the database management tools locally. 

But you should write a readme to show how it can connect to database for debugging. I cannot do it.